docs/content/Auth/Security-Context.md

```diff
@@ -52,26 +61,27 @@ Schema.
 | ```
 <!-- prettier-ignore-end -->
 
-Consider the following example. We want to show orders only for customers who
-own these orders. The `orders` table has a `user_id` column, which we can use to
-filter the results.
+To ensure that users making this query only receive their own orders, define
+`queryRewrite` in the `cube.js` configuration file:
 
 ```javascript
-cube(`Orders`, {
-  sql: `SELECT * FROM public.orders WHERE ${SECURITY_CONTEXT.user_id.filter(
-    'user_id'
-  )}`,
-
-  measures: {
-    count: {
-      type: `count`,
-    },
+module.exports = {
+  queryRewrite: (query, { securityContext }) => {
+    // Ensure `securityContext` has an `id` property
+    if (!securityContext.user_id) {
```

Comment:
I don't think it is best practice to throw an error here. I'd rather return the query.

```js
if (!securityContext.user_id) {
  return query;
}
```

@paveltiunov @igorlukanin wdyt?

Reply:
@keydunov Is it worth also adding a `checkAuth()` property here to do the JWT verification separately, similar to the tests in https://github.com/cube-js/cube.js/pull/2568?